### ISSUE TITLE
v0.25 beta5a - Momiji's First Spell crashes

### ISSUE THREAD

**Issue by Sparen** [ExPorygon/Ephemeral-Unnatural-Balance#27]
As of 09e6571ab9f7ebd34c2b1952804bed8e7d7ee04d, Momiji's first spell begins, then crashes with the following:

```
This variables does not allow to array index operation.
(配列以外にindexを使いました)
Z:/Users/andrewfan/ssh/Ephemeral-Unnatural-Balance/System/Effect.dnh 
Effect.dnh line(行)=257

↓
		case(9.18) { ExplosionRainbowB3(objEnemy,"Water"); }
		case(9.19) { ExplosionRainbowB3(objEnemy,"Sun"); }
		
		case(10) { PPCExplode(objEnemy,eventParam[2]); }
		case(11) { PPCExplodeRainbow(objEnemy); }
	}
```

Found using Test_Stage.dnh in Stage 3 directory

**Comment by ExPorygon** [comment]
I assume you mean Stage.dnh, Test_Stage.dnh does not exist.

**Comment by Sparen** [comment]
Correct - the #Title for Stage.dnh is TestStage so I confused myself

**Comment by ExPorygon** [comment]
Fixed in 648c7e8e81238f58d2510b97c1449434011f2e8c.
